README.md

```diff
@@ -1,2 +1,3 @@
 # fiware-health
 Program to check the functionality of each FIWARE Lab node.
```

Comment:
Will this code be the unique content in this repo? or are we going to have other scripts related with "fiware-health"?